.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,46 @@
+name: Markdownlint
+
+on:
+  pull_request:
+    branches-ignore:
+      - translations
+  push:
+    branches:
+      - main
+
+jobs:
+  see_if_should_skip:
+    runs-on: ubuntu-latest
+
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@36feb0d8d062137530c2e00bd278d138fe191289
+        with:
+          cancel_others: 'false'
+          github_token: ${{ github.token }}
+          paths: '["**/*.md", "!translations/**", "package*.json", ".markdownlint.json", ".github/workflows/markdownlint.yml", ".github/workflows/markdownlint-problem-matcher.json"]'
+          paths_ignore: '["translations/**"]'
+  lint:
+    runs-on: ubuntu-latest
+    needs: see_if_should_skip
+    if: ${{ needs.see_if_should_skip.outputs.should_skip != 'true' }}
+    steps:
+      - uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675
+
+      - uses: actions/cache@d1255ad9362389eac595a9ae406b8e8cb3331f16
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
I think the previous step needs an `id` of "cache" for this to work. 🤔

Reply:
Perhaps this would work as the default ID might match the Action's name? Not sure. Either way, I'd personally like to see an explicit `id` added when referencing it later. 🤷‍♂️

Reply:
OK, I'll see about adding it, but I think this was just copied from the existing Lint job